infoschema/infoschema.go

```diff
@@ -328,16 +330,17 @@ const (
 	codeForeignKeyNotExists = 1091
 	codeWrongFkDef          = 1239
 
-	codeDatabaseExists   = 1007
-	codeTableExists      = 1050
-	codeBadTable         = 1051
-	codeBadUser          = 3162
-	codeColumnExists     = 1060
-	codeIndexExists      = 1831
-	codeMultiplePriKey   = 1068
-	codeTooManyKeyParts  = 1070
-	codeKeyNameDuplicate = 1061
-	codeKeyNotExists     = 1176
+	codeDatabaseExists         = 1007
+	codeTableExists            = 1050
+	codeBadTable               = 1051
+	codeBadUser                = 3162
+	codeColumnExists           = 1060
+	codeReservedColumnConflict = 1066
```

Comment:
According to MySQL [documentation](https://dev.mysql.com/doc/refman/8.0/en/server-error-reference.html), 
> Error number: 1066; Symbol: ER_NONUNIQ_TABLE; SQLSTATE: 42000
Message: Not unique table/alias: '%s'

The code 1000-2000, 3000-4000 is used for MySQL error.
Maybe you should use something TiDB-specific, like 215.